### DISCUSSION TITLE
「追日」飄浮太陽能發電🔅

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#863] [General]
**新聞來源:**
https://buzzorange.com/techorange/2022/11/28/floating-solar-panels-follow-sun/

**新聞大綱:**
近期荷蘭創新的漂浮太陽能發電廠「Proteus」證明了將漂浮太陽能電池板與追日系統結合有助於大幅提升發電效益。
而它的面板每隔幾個小時就會緩慢旋轉，精確地跟踪太陽的路徑，盡可能捕獲更多日光。
當太陽能電池板變得太熱時，發電效益就會下降。因此懸浮在水上可以達到冷卻效果，讓太陽能電池板比陸上同類產品保持更低的溫度，維持一定的發電效率。
我國首座浮動型太陽能光電廠位於屏東大武丁滯洪池上面，於 2016 年 2 月啟用。若將漂浮太陽能發電廠結合追日系統，大幅提升發電效益，或許台灣也可以將水庫作為巨型電池。
**心得:**
我覺得如果台灣能多設立幾個有追日的飄浮太陽能區域的話，感覺可以提升使用綠能的比例，而且現在全球都在以2050年淨零排放作為目標，感覺綠能發展這個市場會很有商機，賺錢錢的同時還能保護地球！

![image](https://user-images.githubusercontent.com/90883526/204467836-22e70c3c-e9bf-435d-a5f1-8346bbe57b46.png)

**Comment by Pietra1226** [comment]
感覺是個很有前景的綠能設施，只是不知道會不會造成近海的生態破壞或是其他連帶影響，希望可以將對環境傷害降到最低。

**Reply by Evan0979** [reply]
欸對,要是創造綠能的同時破壞到環境就有點失去本意了

**Comment by HuangHsinTzu** [comment]
聽起來很不錯欸！
很開心看到許多國內外的團隊努力研究減少碳排的發電方式
但除非政府補助或想辦法壓低價錢
不然沒辦法普及的話，效益也是有限的

**Reply by bonny123865** [reply]
普及感覺是真的有難度，但是如果願意政府或是其他部門有考慮好設立的據點
並投資經費在上面的話，感覺也是能有一些效益!!

**Comment by Kotori0501** [comment]
感覺還不錯耶，畢竟台灣四面環海
只要不要有污染源影響生物健康等等的因素在的話，我認為這個實踐是很不錯的

**Comment by moonnight3503** [comment]
現在環境汙染嚴重，能有更多的方法增加率能是非常好的！

**Comment by a45320614** [comment]
在水庫上建造這種太陽能發電系統真的是將空間利用到極致
如果這個系統能實際應用在台灣
應該不需要再用愛發電了

**Reply by bonny123865** [reply]
而且也能讓傳統發電比例下降，感覺一舉兩得!!

**Comment by Chiway1229** [comment]
雖然看起來非常的理想，但是感覺還是有某些阻力導致無法真的廣泛使用
像是汙染、造價或珍稀資源的缺少等等

**Comment by peggy0329** [comment]
針對2050年淨零探排政策，各國真的都紛紛推出對環境友善的措施!
這是一個蠻有趣的解決方式，但我認為現階段如同前一位同學所說的有點過於理想
除了經費問題
當海面佈滿太陽能發電，雖然透過海水可以降低發電造成的熱能
但感覺反而會對海洋生物多少造成影響
開發者如何在不破壞環境下使用綠能，並取得平衡
我認為是開發者值得思考的問題

**Comment by Evan0979** [comment]
好像向日葵喔

**Comment by ching020721** [comment]
提升綠能的同時希望也能顧及到海洋的環境生態，不然就本末倒置了

**Comment by StevenWanglolz** [comment]
最近教小學伴英文有提到太陽能發電，我一定要把這個科技介紹給他們看!

**Comment by WEI-CHEN-CHUN** [comment]
這樣的方式能提高太陽能的使用效率，但目前太陽能已經占用了不少土地，如果開始入侵水域，可能導致更多問題。

**Comment by williamyen0623** [comment]
二氧化碳的超量排放造成臭氧層破洞、全球暖化，近年來的夏天，我都會思考我才那麼年輕，夏天就熱成這樣，我真的活得到老嗎？
節能減碳是全人類都必須面臨的課題，希望往後有更多像是漂浮太陽能板這種新興的環保發電方式

**Comment by manipulateed** [comment]
的確現在都在想著如何減能減碳或是替代能源，利弊評估更是重要的是事情，追蹤太陽的太陽能板固然大大增加發電效率，但至於水中造成的危害成本會小於帶來的效益嗎

**Comment by CPCpyt** [comment]
這個真的是非常智能的技術，對於緯度較高國家會很需要這樣的技術，
追蹤太陽的位置讓產能最大化，實在令人驚豔！
